OverlayPlugin.Core/MemoryProcessors/AtkStage/FFXIVClientStructs/Data.cs

```diff
@@ -18,6 +18,9 @@ public class Data
         private readonly string yamlFilePath;
         private readonly Dictionary<DataNamespace, ClientStructsData> data = new Dictionary<DataNamespace, ClientStructsData>();
 
+        // @TODO: Is there some way to get this from the module instead?
+        private const long DataBaseOffset = 0x140000000;
```

Comment:
:eyes:

Does this ever change?

Reply:
It hasn't changed since the start of the FFXIVClientStructs project AFAIK. They have this offset hardcoded in the python scripts, so technically we could pull it from that but I didn't want to make this more complicated unless needed.

Reply:
Sounds good to me.  Just didn't know how risky this was.

Reply:
More info on this here:
<https://learn.microsoft.com/en-us/cpp/build/reference/base-base-address?view=msvc-170>

Relevant section:
> The default base address for an EXE file is 0x400000 for 32-bit images or 0x140000000 for 64-bit images.

Basically, it shouldn't change unless SE changes their build process to use a different base address, or to use ASLR.